yaml_specs/modulemd_packager_v2.yaml

```diff
@@ -0,0 +1,255 @@
+                # Use the "buildafter" value to specify that this component
+                # must be be ordered later than some other entries in this map.
+                # The values of this array come from the keys of this map and
+                # not the real component name to enable bootstrapping.
+                buildafter:
+                    - baz
+                # If buildroot is set to True, the packages listed in this
+                # module's buildroot profile will be installed into the
+                # buildroot of any component built in subsequent
+                # buildafter batches, even if that module does not
+                # explicitly include these packages in its BuildRequires.
+                # Optional. Defaults to "false" if not specified.
+                buildroot: false
+                # If srpm-buildroot is set to True, the packages listed in this
+                # module's srpm-buildroot profile will be installed into the
+                # buildroot when performing the buildSRPMfromSCM step in
+                # subsequent buildafter batches.
+                # Optional. Defaults to "false" if not specified.
+                srpm-buildroot: false
+            # baz has no extra options
+            baz:
+                rationale: This one is here to demonstrate other stuff.
+            xxx:
+                rationale: xxx demonstrates arches and multilib.
+                # xxx is only available on the listed architectures.
+                # Includes specific hardware architectures, not families.
+                # See the data.arch field for details.
+                # Optional, defaults to all available arches.
+                arches: [i686, x86_64]
+                # A list of architectures with multilib
+                # installs, i.e. both i686 and x86_64
+                # versions will be installed on x86_64.
+                # Includes specific hardware architectures, not families.
+                # See the data.arch field for details.
+                # Optional, defaults to no multilib.
+                multilib: [x86_64]
+                # This package requires a particular version of 'xyz' to be
+                # present in order to build correctly, so we inform the build
+                # system not to build this until 'xyz' has completed
+                # successfully.
+                buildafter:
+                    - xyz
+            xyz:
+                rationale: xyz is a bundled dependency of xxx.
+        # Module content of this module
+        # Included modules are built in the shared buildroot, together with
+        # other included content.  Keys are module names, values additional
+        # component information.  Note this only includes components and their
+        # properties from the referenced module and doesn't inherit any
+        # additional module metadata such as the module's dependencies or
+        # component buildopts.
```

Comment:
Since the packager spec drops `buildopts`, should this reference to `buildopts` also be dropped?

Reply:
Agreed